app/views/energy_tariffs/energy_tariffs/show.html.erb

```diff
@@ -24,12 +83,49 @@
 
   <br/>
 
-  <h4 id="standing-charges">
-    <%= t('schools.user_tariffs.show.standing_charges') %>
-    <%= link_to t('common.labels.edit'), energy_tariffs_path(@energy_tariff, [:energy_tariff_charges]), class: 'btn' %>
-  </h4>
+  <div class="d-flex justify-content-between" id="standing-charges">
+    <div>
+      <h4><%= t('schools.user_tariffs.show.standing_charges') %></h4>
+    </div>
+    <div>
+      <%= link_to t('common.labels.edit'), energy_tariffs_path(@energy_tariff, [:energy_tariff_charges]), class: 'btn' %>
+    </div>
+  </div>
 
   <%= render 'energy_tariffs/energy_tariffs/charges_table', energy_tariff: @energy_tariff, allow_edits: false %>
+  <br />
+
+  <% if current_user.admin? %>
+    <div class="d-flex justify-content-between">
+      <div>
+        <h4><%= t('schools.user_tariffs.show.notes_html') %></h4>
+      </div>
+    </div>
+    <div>
+      <table class="table table-charges">
+        <tbody>
+          <tr>
+            <td class="description"><%= t('schools.user_tariffs.show.created_at') %></td>
+            <td class="value"><%= @energy_tariff.created_at&.to_s(:es_compact) %></td>
```

Comment:
This `es_compact` doesn't seem to work? Viewing a tariff I see: "2023-08-01 13:55:41 UTC" not `%d/%m/%Y %H:%M`